llama_stack/providers/utils/telemetry/sqlite_trace_store.py

```diff
@@ -166,14 +173,31 @@ async def get_span_tree(
                 return spans_by_id
 
     async def get_trace(self, trace_id: str) -> Trace:
-        query = "SELECT * FROM traces WHERE trace_id = ?"
+        query = """
```

Comment:
why has that query changed?

Reply:
to fix https://github.com/meta-llama/llama-stack/pull/2814